specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2018-02-14/keyvault.json

```diff
@@ -0,0 +1,1359 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-02-14-preiview",
```

Comment:
>preiview [](start = 27, length = 8)

preview :) ?

Reply:
All three files have this typo

---
In reply to: [168832029](https://github.com/Azure/azure-rest-api-specs/pull/2509#discussion_r168832029) [](ancestors = 168832029)